vulnerabilities/importers/istio.py

```diff
@@ -210,10 +174,10 @@ def process_file(self, path):
         return advisories
 
     def get_data_from_md(self, path):
-        """Return a mapping of vulnerability data from istio . The data is
-        in the form of yaml_lines inside a .md file.
+        """Return a mapping of vulnerability data from istio. The data is
+        in the form of yaml objects found inside front matter of the .md file.
```

Comment:
You are telling too much about how the sausage is made here in your documentation.
 `Return a mapping of vulnerability data extracted from an advisory.` may be plenty enough.

Reply:
fixed